src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRow1099Processor.java

```diff
@@ -925,7 +927,13 @@ private void checkForGeneralExclusionsAndFindTaxBox(ResultSet rs, Transaction109
         int idx;
         
         // Find tax box.
-        if (getPaymentReason1099BoxService().isPaymentReasonMappedTo1099Box(paymentReasonCodeP.value)) {
+        if (getDocumentType1099BoxService().isDocumentTypeMappedTo1099Box(docTypeP.value)) {

```

Comment:
This has to do with how I had implemented the TAX module at the time, where the tax processor classes were intended to manipulate those fields directly and the superclass was intended to use the appropriate getValue() methods instead. I suppose getValue() could work in this case due to it being a string value, but doing the field access would be consistent with the rest of the code.

Perhaps the implementation of the TAX module can be revisited at a later date to try and make this cleaner, but that would be beyond the scope of this PR.